### ISSUE TITLE
Tensorborad show Learning rate on every batch

### ISSUE THREAD

**Issue by yajunDai** [keras-team/keras#17130]
First: 
```
class LRTensorBoard(TensorBoard):
    # add other arguments to __init__ if you need
    def __init__(self, log_dir, **kwargs):
        super().__init__(log_dir=log_dir, **kwargs)

    def on_epoch_end(self, epoch, logs=None):
        logs = logs or {}
        logs.update({'lr': K.eval(self.model.optimizer.lr)})
        super().on_epoch_end(epoch, logs)

    def on_batch_end(self, batch, logs=None):
        logs = logs or {}
        logs.update({'lr': K.eval(self.model.optimizer.lr)})
        super().on_batch_end(batch, logs)
```
And Then when train:
```
callbacks = [
            LRTensorBoard(log_dir='./logs', histogram_freq=1, update_freq="batch")]
model.fit(..., callbacks=callbacks)
```
Last:
![image](https://user-images.githubusercontent.com/69563639/194700777-e52d0720-0f6d-4f63-94c6-fd07345c1466.png)

there's not batch_lr. why?

Thanks

**Comment by tilakrayal** [comment]
@yajunDai,
To expedite the trouble-shooting process, could you please provide a complete code snippet and the TensorFlow version you are using. Thank you!

**Comment by tilakrayal** [comment]
@yajunDai,
Code shared is full of [indentation errors](https://colab.research.google.com/gist/tilakrayal/0a3c6f1a9f752c60f7b51b82cf5ff2de/untitled681.ipynb), please share a colab gist with issue reported or simple stand alone indented code with all dependencies such that we can replicate the issue reported. Thank you!
